packages/dataclass-editor/src/components/dataclass/detail/FieldDetailContent.tsx

```diff
@@ -26,28 +39,36 @@ export const FieldDetailContent = () => {
       <BasicField label='Type'>
         <Input value={field.type} onChange={event => handleFieldPropertyChange('type', event.target.value)} />
       </BasicField>
```

Comment:
To hold it constistent, I would suggest to pack the name, type and comment also inside of a collapsible